docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,19 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service Rollback to Zero (CB90067)
+
+*Background*: A service has been forced to roll back to zero.
+This is bad because it means the service is attempting to resynchronize the entirety of the data set.
+
+*Condition*: A rollback to zero is seen in the `memcached` logs.
+
+*Remediation*: This is typically a symptom of another problem, you may ignore this unless you notice any other issues with your cluster.
```

Comment:
Hm, not so sure about this - even if this is just a symptom, indexing having to re-index the entirety of the dataset (possibly terabytes of data) could be problematic, so I wouldn't recommend you ignore it. Perhaps monitor your cluster health or something like that?

Reply:
How does "This is typically a symptom of another problem, you should monitor your cluster closely for any further issues. If you experience any, then please Contact Couchbase Technical Support." sound?